src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 2.0.26
 ++++++
+* vmss instance update: support attach/detach disks on an individual instance
```

Comment:
This should be for the next version of the CLI, no?

Reply:
You are right. Should be the next one.